src/components/Verification/VerificationButton.jsx

```diff
@@ -0,0 +1,298 @@
+    const newVerification = {
+      verified: false,
+      last_modifier: name || 'phlask_app',
+      last_modified: new Date().toISOString()
+    };
+    const newResource = {
+      ...resource,
+      verification: newVerification
+    };
+    updateFirebaseEntry(newResource);
+  }, []);
+
+  const markAsInactive = useCallback(() => {
+    const newVerification = {
+      verified: false,
+      last_modifier: name || 'phlask_app',
+      last_modified: new Date().toISOString()
+    };
+    const newResource = {
+      ...resource,
+      status: 'HIDDEN',
+      verification: newVerification
+    };
+    updateFirebaseEntry(newResource);
+  }, []);
+
+  if (!resource) {
+    return;
+  }
+
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        right: '10px',
+        bottom: '10px',
+        backgroundColor: resource.verification.verified ? 'Green' : 'Tomato',
+        padding: '5px',
+        borderRadius: '5px',
+        color: 'white',
+        fontSize: '12px',
+        fontWeight: 'bold',
+        textAlign: 'center'
+      }}
+    >
+      <div onClick={() => setIsModalOpen(true)} style={{ cursor: 'pointer' }}>
+        {resource.verification.verified ? 'VERIFIED' : 'UNVERIFIED'}
+      </div>
+      <Dialog open={isModalOpen} onClose={closeModal}>
+        <div
+          style={{
+            width: '300px',
+            padding: '16px'
+          }}
+        >
+          {!isAdmin && (
+            <div>
+              <p>
+                Enter the admin password to update the verified status of this
+                resource. Also enter your name.
+              </p>
+              <Input
+                placeholder="Your name"
+                style={{ width: '100%', marginBottom: '10px' }}
+                type="text"
+                autoComplete="disabled"
+                value={name}
+                onChange={e => {
+                  setName(e.target.value);
+                }}
+              />
+              <Input
+                placeholder="The admin password"
+                style={{ width: '100%', marginBottom: '10px' }}
+                type="password"
+                autoComplete="disabled"
+                value={password}
+                onChange={e => {
+                  setPassword(e.target.value);
+                  setLoginError('');
+                }}
+              />
+              {loginError && <p style={{ color: 'red' }}>{loginError}</p>}
+              <div
+                style={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  gap: '10px'
+                }}
+              >
+                <Button
+                  onClick={closeModal}
+                  style={{
+                    flex: 1,
+                    backgroundColor: 'gray',
+                    color: 'white',
+                    borderRadius: '4px',
+                    padding: '4px'
+                  }}
```

Comment:
Similar to other mui components, we should use the `sx` attribute here in place of `style`

Reply:
Fixed!